src/runtime/InternalLoader.js

```diff
@@ -158,7 +158,7 @@ class BundledCodeUnit extends CodeUnit {
     var normalizedNames =
         this.deps.map((name) => this.loader_.normalize(name));
     var module = this.execute.apply(Reflect.global, normalizedNames);
-    System.set(this.normalizedName, module);
+    $traceurRuntime.setModule(this.normalizedName, module);
```

Comment:
I see. The problem is that we need to bootstrap something using a script.

How about a middle ground for now:

```js
const {ModuleStore} = $traceurRuntime;
...
ModuleStore.set(this.normalizedName, module);
```

?